qubes/qmemman/__init__.py

```diff
@@ -57,6 +57,10 @@ def __init__(self):
         self.log.debug('SystemState()')
 
         self.domdict = {}
+        self.xc = None
+        self.xs = None
+
+    def init(self):
```

Comment:
I don't think I'm against this change, but can you provide the reason (preferably in the commit message)?